MekHQ/src/mekhq/campaign/Kill.java

```diff
@@ -41,15 +41,19 @@ public class Kill {
     private LocalDate date;
     private String killed;
     private String killer;
+    private int missionId;
+    private int scenarioId;
```

Comment:
The primitive 'int' is assigned a value of 0 by default

https://stackoverflow.com/questions/27211713/why-is-an-integer-variable-assigned-null-by-default

<img width="713" alt="image" src="https://github.com/MegaMek/mekhq/assets/103902653/404a30f7-1885-4e7d-ad83-5b828ee0dfb5">

Reply:
:see_no_evil:

Reply:
Don't worry, I only discovered this by complete accident. But I rely on it a lot here, because it allows for easy handling of older saves with null `missionID` and `scenarioID` values